src/helper/Redux/actions/reminder.action.js

```diff
@@ -6,9 +6,28 @@ import {
 } from '../types/reminder.types';
 
 //TODO: fetch reminders
-export const fetchReminder = () => {
+export const fetchReminder = (reminders = null) => {
   return {
     type: FETCH_REMINDER,
+    payload: {
+      reminders,
+    },
+  };
+};
+export const fetchReminderSucess = (status = false) => {
+  return {
+    type: FETCH_REMINDER,
+    payload: {
+      status: true,
+    },
+  };
+};
+export const fetchReminderFailure = (status = false) => {
+  return {
+    type: FETCH_REMINDER,
+    payload: {
+      status: true,
+    },
```

Comment:
here I change the value of our status indicators, this is the good direction and data flow.